app/assets/javascripts/modules/track-results.js

```diff
@@ -0,0 +1,48 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function TrackResults ($module) {
+    this.$module = $module
+    this.flowName = this.$module.dataset.flowName
+    this.page = document.location.pathname
+  }
+
+  TrackResults.prototype.init = function () {
+    this.trackCompletedFlow()
+    this.trackResultLinks()
+  }
+
+  TrackResults.prototype.trackCompletedFlow = function () {
+    var options = {
+      label: this.flowName,
+      nonInteraction: true,
+      page: this.page
+    }
+    GOVUK.analytics.trackEvent('Smart Answer', 'Completed', options)
+  }
+
+  TrackResults.prototype.trackResultLinks = function () {
```

Comment:
Might be good to reflect in the naming of this method that this only tracks links on the same domain (or internal links). Just to prevent confusion that this doesn't send events for external links (that comes from static).

Reply:
Good point – renamed to `trackInternalLinks` for clarity.